horse/recommendation/smart.py

```diff
@@ -0,0 +1,92 @@
+from collections import defaultdict
+
+from .base import RecommendationService
+
+
+class SmartRecommendationService(RecommendationService):
+    user_similarity_treshold = 2
+    movie_likes_fraction_exp = 0.5
```

Comment:
Hm, in fact those attributes can be set on a per-instance basis. I'll migrate them to default `__init__` values.